lib/dependency_validator.dart

```diff
@@ -23,15 +25,25 @@ export 'src/constants.dart' show commonBinaryPackages;
 
 /// Check for missing, under-promoted, over-promoted, and unused dependencies.
 void run({
-  List<String> excludedDirs = const [],
-  bool fatalDevMissing = true,
-  bool fatalMissing = true,
-  bool fatalOverPromoted = true,
-  bool fatalPins = true,
-  bool fatalUnderPromoted = true,
-  bool fatalUnused = true,
-  List<String> ignoredPackages = const [],
+  List<Glob> excludes,
```

Comment:
this is technically a breaking change (I replaced `excludedDirs` with `excludes`), but if we want to release this as a minor change to start, we could keep `excludedDirs` and deprecate it